collada_urdf/CMakeLists.txt

```diff
@@ -48,13 +48,33 @@ endif()
 find_package(Boost REQUIRED COMPONENTS system filesystem program_options)
 include_directories(${Boost_INCLUDE_DIR})
 
+# We don't build anything here, but in order to export the right
+# build flags in catkin_package(), we still have to find Eigen3 here.  Note
+# that this is somewhat complicated because of our need to support Ubuntu
+# all the way back to saucy.  First we look for the Eigen3 cmake module
```

Comment:
[REP 3](http://www.ros.org/reps/rep-0003.html#kinetic-kame-may-2016-may-2021) says kinetic targets xenial and above. [libeigen3-dev](https://packages.ubuntu.com/search?keywords=libeigen3-dev) is in trusty and above. I think this block can be replaced with `find_package(Eigen3 REQUIRED)`

Reply:
Ah, good point.  It looks like Kinetic used to support Wily, but we've long stopped worrying about that.  Fixed in 4edadbf